.github/workflows/import-to-elasticsearch.yml

```diff
@@ -94,7 +94,9 @@ jobs:
       fail-fast: false
       matrix:
         group:
-          - "manual"
+          - "github-atom"
+          - "github-ntoz"
+          - "other"
```

Comment:
should these match up with the toml file names, as `manual` did, and `github-atom` does?
because `github-ntoz` seems missing so far, whereas `other` seems not to match `others`?

Reply:
> should these match up with the toml file names, as `manual` did, and `github-atom` does?

Correct!

> because `github-ntoz` seems missing so far, whereas `other` seems not to match `others`?

Oops, good eyes, forgot to commit/rename ;) . Now updated.